src/cli.ls

```diff
@@ -52,7 +52,7 @@ program
     # options about LaTeX and style
     .option '-c, --class <class>',      'set a default documentclass for documents without a preamble', 'article'
     .option '-m, --macros <file>',      'load a JavaScript file with additional custom macros'
-    .option '-s, --style <url>',        'specify an additional style sheet to use (can be repeated)', addStyle
+    .option '-s, --stylesheet <url>',   'specify an additional style sheet to use (can be repeated)', addStyle
 
```

Comment:
good idea, but then you also need to change line 78

Reply:
Okay, wait a minute. I'll update it soon.